kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -0,0 +1,200 @@
+# Creating Oracle DB service and RCU Schema for a Fusion Middleware domain
+
+This sample demonstrates how to create an Oracle DB service on Kubernetes cluster and create RCU schema on the Oracle DB being used by a Fusion Middleware domain.  
+
+The directory contains the following sample scripts to start an Oracle DB service in default namespace, stop the Oracle DB service, create an RCU schema and drop the RCU schema
```

Comment:
This is confusing, I would turn this into a bulleted list so it is clearer that these are four separate examples